contracts/QuestFactory.sol

```diff
@@ -89,10 +89,9 @@ contract QuestFactory is Initializable, LegacyStorage, OwnableRoles, IQuestFacto
                                MODIFIERS
     //////////////////////////////////////////////////////////////*/
 
-    modifier checkQuest(string memory questId_, address rewardTokenAddress_) {
+    modifier checkQuest(string memory questId_) {
         Quest storage currentQuest = quests[questId_];
         if (currentQuest.questAddress != address(0)) revert QuestIdUsed();
-        if (!rewardAllowlist[rewardTokenAddress_]) revert RewardNotAllowed();
```

Comment:
Should we also set a comment on line 44 that `mapping(address => bool) public rewardAllowlist;` is not used